editor-components/modal/notificationmodal2/notificationmodal2.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import { BaseModal } from "../../EditorComponent";
+import styles from "./notificationmodal2.module.scss";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+
+class NotificationModal2 extends BaseModal {
+  constructor(props?: any) {
+    super(props, styles);
+
+    this.addProp({
+      type: "icon",
+      key: "icon",
+      displayer: "Icon",
+      value: "PiBellRingingThin",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "notification-title",
+      displayer: "Title",
+      value: "Notifications",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "displayer-message",
+      displayer: "Description",
+      value: "Receive an email every time a new lesson is added to the website.",
+    });
+
+    this.addProp({
+      type: "string",
+      key: "e-mail",
+      displayer: "E-mail",
+      value: "Your e-mail",
+    });
+
+    this.addProp({
+      type: "object",
+      key: "button",
```

Comment:
This does not need to be an object. User is only going to enter button text. It can be just string.